package.json

```diff
@@ -91,6 +95,8 @@
     "@types/sketchapp": "^1.0.0",
     "codelyzer": "~4.5.0",
     "conventional-changelog-cli": "^2.0.12",
+    "cypress": "^3.1.5",
+    "cypress-testing-library": "^2.3.6",
```

Comment:
why do we need this extra dep?

Reply:
This provides nice selectors that don't rely on the html structure of the page, as it is been used https://github.com/xlayers/xlayers/pull/285/files#diff-7171ec2293184cca45c912ee69d44de2R7